saleor/graphql/order/mutations/orders.py

```diff
@@ -560,3 +561,35 @@ def perform_mutation(cls, _root, info, **data):
             cls.get_node_or_error(info, data["id"], only_type=Invoice)
         )
         return super().perform_mutation(_root, info, **data)
+
+
+class UpdateInvoice(ModelMutation):
+    invoice = graphene.Field(Invoice, description="An updated invoice.")
```

Comment:
@maarcingebala  This field is generated by `ModelMutation`?

Reply:
It does! Didn't knew about it, removing.